server/app/routes/tags/index.js

```diff
@@ -33,6 +33,7 @@ router.get('/:id', function(req, res, next){
 });
 
 router.post('/', function(req, res, next){
+    req.body.title = req.body.title.toLowerCase()
```

Comment:
If you want your titles to always be saved as lowercase, this should be set in the model, not in the route. @noeladd